content/docs/engine-dev/scenepipeline/addingscenefiles.md

```diff
@@ -0,0 +1,43 @@
+---
+linkTitle: Adding Scene Files
+title: Adding Scene Files
+description: Describes the steps to take to add support for additional scene file formats.
+
+weight: 100
+---
+
+## Overview
+
+O3DE uses the [Open Asset Importer Library](./openassetimporter) to load scene files from a variety of scene formats, including files in the `fbx` format.
+
+This page covers the steps to take to enable additional scene file formats in the scene pipeline.
+
+## Enabling a scene format supported by [Open Asset Importer Library](./openassetimporter)
+
+### Understand the feature set of the O3DE scene pipeline, and the file format you are enabling
+
+There is a lot of variance in what information scene files of different formats can support. For example, the `stl` format does not contain information like UVs for texture mapping. You will want to read through the available documentation for the scene file format you are enabling before moving to the next step.
+
+Understanding what the format you are enabling supports and what the scene pipeline supports will help you figure out your approach to enabling the format, especially in relation to testing, both manual and automated, that will be done before finishing this work. You can read more about the scene pipeline in the [Scene Pipeline](../../user-guide/assets/scene-pipeline) and [Scene Settings](../../user-guide/assets/scene-settings) documentation for O3DE.
+
+Finally, you'll want to understand the current support level for the format you are enabling, in the [Open Asset Importer Library](./openassetimporter) itself. Scene files often have many sub versions, for example `fbx` is not a single static file format, but it has been extended and changed throughout the years, and [Open Asset Importer Library](./openassetimporter) best supports 2011, 2012, and 2013 versions of the `fbx` file format.
```

Comment:
```suggestion
Finally, you'll want to understand the current support level for the format you are enabling, in the [Open Asset Importer Library](./openassetimporter) itself. Scene files often have many sub versions, for example `fbx` is not a single static file format, but it has been extended and changed throughout the years, and [Open Asset Importer Library](./openassetimporter) supports 2011, 2012, and 2013 versions of the `fbx` file format.
```

Reply:
IMHO `bests` by itself is not helpful. You could mention it officially supports versions, or add line to say that AssImp can import other versions but may not correctly import all features etc.